components/Article/InlineCode/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import InlineCode from './index';
+import type { FC } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+type InlineCodeArgs = {
+  code: string;
+};
+
+const meta: Meta<FC<InlineCodeArgs>> = {
+  component: InlineCode as FC<InlineCodeArgs>,
+  decorators: [
+    (Story, context) => (
+      <InlineCode>
+        <code>{context.args.code}</code>
+      </InlineCode>
+    ),
+  ],
+};
+
+export default meta;
```

Comment:
This export should be below everything and the naming of the export should be consistent as from the template on the CONTRIBUTING guidelines (same for the other story)